src/CommaSeparatedJsonParser.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class CommaSeparatedJsonParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (!empty($value) && $value[0] !== '[' && $value[strlen($value) - 1] !== ']') {
```

Comment:
"So perhaps we just decorate it with square brackets anyway" <- IMO that's a good solution :)